### ISSUE TITLE
Check compatibility with tns-core-modules@3.4.0 and update peerDependencies

### ISSUE THREAD

**Issue by hettiger** [bradmartin/nativescript-cardview#75]
```bash
$ tns plugin add nativescript-cardview
npm WARN nativescript-cardview@2.0.4 requires a peer of tns-core-modules@~3.3.0 but none is installed. You must install peer dependencies yourself.
```

**Comment by akaco** [comment]
getting same warning on New project 
nativescript-cardview@2.0.4 requires a peer of tns-core-modules@~3.3.0 but none is installed. You must install peer dependencies yourself.
and not work 

 "dependencies": {
    "@angular/animations": "~5.1.0",
    "@angular/common": "~5.1.0",
    "@angular/compiler": "~5.1.0",
    "@angular/core": "~5.1.0",
    "@angular/forms": "~5.1.0",
    "@angular/http": "~5.1.0",
    "@angular/platform-browser": "~5.1.0",
    "@angular/platform-browser-dynamic": "~5.1.0",
    "@angular/router": "~5.1.0",
    "nativescript-angular": "~5.1.0",
    "nativescript-cardview": "2.0.4",
    "nativescript-pro-ui": "~3.3.0",
    "nativescript-theme-core": "~1.0.4",
    "reflect-metadata": "~0.1.10",
    "rxjs": "~5.5.5",
    "tns-core-modules": "~3.4.0",
    "zone.js": "~0.8.18"
  },

**Comment by bradmartin** [comment]
I published 2.0.5 just now with `^` on the peerDep version with 3.3.0 min so now it should be fine with 3.3+. Let me know of any issues and I'll look into just removing it. I don't think it's necessary anymore to have it, it was at one time during the NS 2x to 3x
